elements/rh-secondary-nav/rh-secondary-nav.ts

```diff
@@ -418,6 +434,9 @@ export class RhSecondaryNav extends LitElement {
       this._mobileMenuExpanded = false;
     } else {
       this._mobileMenuExpanded = true;
+      if (this.colorPalette === 'darker') {
```

Comment:
Not sure if we necessarily need this check though it would limit the number of times this value is updated potentially.

Reply:
You are correct about this check happening too often.   The need for that check is due to initialization in mobile view not setting the value for the slotted cta.   After a change to the screen size it would reset, but not on initial load.

I corrected this in e1f98ed90a14a8c3ae2fb04f860c05d7159bc4ad by moving that check to `firstUpdated()`